nepal_compliance/setup/uninstall.py

```
@@ -0,0 +1,82 @@
+
+        if company_name == "Test Pvt Ltd":
+            safe_delete("Company", company_name)
+        
+        if frappe.db.exists("IRD Certification"):
+            settings = frappe.get_doc("IRD Certification")
+            settings.test_data_created = 0
+            settings.save(ignore_permissions=True)
+
+    except Exception as e:
+        frappe.db.rollback()
+        frappe.logger().error(f"Cleanup failed for company {company_name}: {str(e)}")
```

Ruff (0.14.3)

Use explicit conversion flag

Replace with conversion flag

(RUF010)